templates/prometheus.yaml

```diff
@@ -0,0 +1,294 @@
+apiVersion: v1
```

Comment:
I would move the namespace into a separate file - it's in two locations

Reply:
I hope I handled this in an acceptable way